src/test/regress/sql/insert_select_into_local_table.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE dist_table (a INT, b INT, c TEXT);
```

Comment:
Let's put all these things in a schema, so it does not conflict: 
```sql
CREATE SCHEMA insert_select_into_local_table;
SET search_path TO insert_select_into_local_table;
```

Cleanup can then easily be done like this:
```sql
\set VERBOSITY terse
DROP SCHEMA insert_select_into_local_table CASCADE;
```

It also makes sure that this test does not accidentally conflict with any tests run concurrently with it.